backend/dataall/core/environment/services/environment_service.py

```diff
@@ -463,6 +487,20 @@ def remove_group(uri, group):
                 resource_uri=environment.environmentUri,
                 resource_type=Environment.__name__,
             )
+
+            # if group is no longer invited to organization or environments inside it, remove GET_ORGAISATION policy
```

Comment:
nit: type in comment `remove GET_ORGANIZATION policy`

Reply:
done